Scrumdinger/Views/ScrumsView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ScrumsView.swift
+//  Scrumdinger
+//
+//  Created by Life Boss on 7/16/23.
+//
+
+import SwiftUI
+
+struct ScrumsView: View {
+    let scrums: [DailyScrum]
+    
+    var body: some View {
+        List(scrums) { scrum in
+            CardView(scrum: scrum)
+                .listRowBackground(scrum.theme.mainColor)
```

Comment:
☝️ Note that this attribute is different from `background` which only affects a sub-view of the row.